quickwit-doc-mapper/src/tokenizers.rs

```diff
@@ -17,14 +17,105 @@
 // You should have received a copy of the GNU Affero General Public License
 // along with this program. If not, see <http://www.gnu.org/licenses/>.
 
+use regex::Regex;
 use once_cell::sync::Lazy;
-use tantivy::tokenizer::{RawTokenizer, RemoveLongFilter, TextAnalyzer, TokenizerManager};
+use tantivy::tokenizer::{RawTokenizer, RemoveLongFilter, TextAnalyzer, TokenizerManager, Token, TokenStream, Tokenizer, BoxTokenStream};
+use std::str::CharIndices;
+
+/// Tokenize the text without splitting on ".", "-" and "_" in numbers.
+#[derive(Clone)]
+pub struct CustomTokenizer;
```

Comment:
`CustomTokenizer` is too general, it's a tokenizer for logs, so let's call it "LogTokenizer".